xbmc/test/CMakeLists.txt

```diff
@@ -1,11 +1,12 @@
 set(SOURCES TestBasicEnvironment.cpp
             TestCueDocument.cpp
+            TestDateTime.cpp
+            TestDateTimeSpan.cpp
```

Comment:
I assume these are existing files that were mistakenly not included? Because "add some tests" doesn't mention that other tests were missing before.

Reply:
it is just an alpha sort.